blockstore/src/main/java/brooklyn/location/blockstore/effectors/ExtraHddBodyEffector.java

```diff
@@ -107,7 +107,7 @@ public MountedBlockDevice call(ConfigBag parameters) {
             JcloudsLocationCustomizer customizer = getCustomizerForCloud(provider, locationCustomizerFields);
             customizer.customize(machine.getParent(), machine.getParent().getComputeService(), machine);
 
-            return getMountedBlockDevice(customizer);
+            return ((brooklyn.location.blockstore.NewVolumeCustomizer) customizer).getMountedBlockDevice();
```

Comment:
Make `getCustomizerForCloud` return `NewVolumeCustomizer` and remove the cast here.